Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorLogicalEffect.py

```diff
@@ -31,7 +33,7 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Apply logical operators or combine segments<br>. Available operations:<p>
+        return _("""<html>Apply logical operators or combine segments<br>. Available operations:<p>
```

Comment:
```suggestion
        return _("""<html>Apply logical operators or combine segments.<br>Available operations:<p>
```

Reply:
See above, this placement of the `<br>` is intentional.